com.roundabout_host.Izvor.yaml

```diff
@@ -0,0 +1,35 @@
+id: com.roundabout_host.Izvor
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: izvor
+finish-args:
+  - --share=network
+  - --filesystem=host
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  # flatpak-pip-generator
+  - pypi-dependencies.json
+  - name: izvor
+    buildsystem: simple
+    build-commands:
+      - unset GTK_MODULES
+      - mkdir -p /app/bin /app/share/izvor
+      - cp -r ./src/* /app/share/izvor/
+      - chmod +x /app/share/izvor/__init__.py
+      - ln -s /app/share/izvor/__init__.py /app/bin/izvor
+      - mkdir -p /app/lib/python3.12/site-packages/izvor_utils
+      - cp -r ./izvor_utils /app/lib/python3.12/site-packages/izvor_utils/
+      - mkdir -p /app/share/icons/hicolor/scalable/apps
+      - cp com.roundabout_host.Izvor.svg /app/share/icons/hicolor/scalable/apps/
+      - install -Dm644 com.roundabout_host.Izvor.metainfo.xml /app/share/metainfo/com.roundabout_host.Izvor.metainfo.xml
+      - mkdir -p /app/share/applications
+      - cp com.roundabout_host.Izvor.desktop /app/share/applications/
```

Comment:
```suggestion
      - install -Dm0644 com.roundabout_host.Izvor.desktop -t /app/share/applications/
```

Reply:
can you please stop resolving comments without doing or answering anything? we aren't here to chase every thread...